test/runtests.jl

```diff
@@ -8,7 +9,7 @@ using Memento.TestUtils: @test_log, @test_nolog
 using Mocking
 using Test
 
-using AWS.AWSExceptions: AWSException
+@service Batch
```

Comment:
Is this needed if everything is done through `AWSBatch.Batch`?

Reply:
Good catch! I was testing earlier and mocking did not like when I set my patch up as `@patch Batch.X`